views/fe-utils/siteTooltipContent.js

```diff
@@ -0,0 +1,30 @@
+import { TOTAL_LABEL, NA } from '../../constants/vars'
```

Comment:
should this be an `index` file within `constants`? These are all going to be variables that get exported.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/191/commits/830a204d6e575727759e922645ae04fe835fda58